Gems/EMotionFX/Code/Tests/UI/SkeletonOutlinerTestFixture.cpp

```diff
@@ -0,0 +1,172 @@
+                model->index(0, 0, indices[0]),
+                Qt::DisplayRole,
+                QVariant::fromValue(subLevelLabel),
+                    -1,
+                Qt::MatchRecursive);
+        }
+        //  check indices
+        if (indices.size() < 1)
+        {
+            EXPECT_GT(indices.size(), 0) << "Label not found";
+            return;
+        }
+        EXPECT_EQ(indices.size(), 1) << "Label is not eindeutig";
+
+        //  click first index
+        auto index = indices[0];
+        treeView->selectionModel()->select(index, QItemSelectionModel::Select | QItemSelectionModel::Rows);
+        treeView->clicked(index);
+    }
+
+    void SkeletonOutlinerTestFixture::ShowJointPropertyWidget()
+    {
+        auto* widget = GetJointPropertyWidget();
+        auto* mw = new QMainWindow;
+        mw->setFixedHeight(900);
+        mw->layout()->addWidget(widget);
+        widget->setFixedHeight(800);
+
+        mw->show();
+
+        // while (true)
+        QApplication::processEvents();
+    }
+
+    /*
+        Test Cases
+     */
+    TEST_F(SkeletonOutlinerTestFixture, AddClothCollider)
+    {
+        SetUpPhysics();
+
+        // Find the 3rd joint after the RootJoint in the TreeView and select it
+        SelectIndexes(m_indexList, m_treeView, 3, 3);
+
+        AddColliderViaAddComponentButton("Add Cloth Collider", "Sphere");
+
+        ShowJointPropertyWidget();
+
+        EXPECT_TRUE(ColliderHelpers::NodeHasClothCollider(m_indexList[3]));
+    }
+
+    TEST_F(SkeletonOutlinerTestFixture, ChangeClothColliderValue)
+    {
+        SetUpPhysics();
+
+        AddColliderViaAddComponentButton("Add Cloth Collider", "Capsule");
+
+        // Check the node is in the ragdoll
+        EXPECT_TRUE(ColliderHelpers::NodeHasClothCollider(m_indexList[3]));
+
+        // get the widget
+        //auto* widget = GetJointPropertyWidget()->findChild<ClothJointWidget*>("EMotionFX.ClothJointWidget");
+        auto* widget = GetJointPropertyWidget();
+
+        // get a value widget
+        auto propertyEditor = widget->findChild<AzToolsFramework::ReflectedPropertyEditor*>("PropertyEditor");
+
+        // Get list of all PropertyRowWidgets (and their InstanceDataNodes)
+        const auto list = propertyEditor->GetWidgets();
+        ASSERT_GT(list.size(), 0) << "Did not find any PropertyRowWidgets";
+
+        // Look for PropertyRowWidget for "Name"
+        AzToolsFramework::PropertyRowWidget* propertyRow = nullptr;
+        for (const auto& item : list)
+        {
+            if (item.second->objectName() == "Height")
+            {
+                propertyRow = item.second;
+            }
+        }
+
+        // change it
+        auto *lineEdit = static_cast<AzToolsFramework::PropertyDoubleSpinCtrl*>(propertyRow->GetChildWidget());
+        //AZ_Assert(lineEdit) << "Did not find Editing handle";
+        lineEdit->setValue(3.89);
+        lineEdit->editingFinished();
+
+        // watch the world burn
```

Comment:
The tone of the comments is overly wild :) 
Just change it to something like "make sure propertyWidget are created correctly" or something. Also there should be something to actually check if the propertyWidget is created correctly. Maybe check the widget amount or something?